src/krisi/evaluate/evaluate.py

```diff
@@ -2,21 +2,21 @@
 
 import numpy as np
 import pandas as pd
+from sklearn.metrics import mean_absolute_error, mean_squared_error
 from statsmodels.tsa.stattools import acf, pacf, q_stat
 
-from krisi.evaluate.scorecard import SampleTypes, ScoreCard
+from krisi.evaluate.scorecard import Metric, SampleTypes, ScoreCard
 
 
 def metric_hoc(func: Callable, *args, **kwargs) -> Callable:
```

Comment:
technically, this is called partial application of a function, not higher order component (that's React specific)

Reply:
What would be a shorter word for it?